presidio-analyzer/pyproject.toml

```diff
@@ -27,7 +27,9 @@ dependencies = [
     "regex",
     "tldextract",
     "pyyaml",
-    "phonenumbers (>=8.12,<10.0.0)"
+    "phonenumbers (>=8.12,<10.0.0)",
+    "azure-health-deidentification (>=1.0.0,<2.0.0)",
```

Comment:
Can you please add all the azure dependencies as optional? We'd like to keep the base package as simple as possible. You already have the ahds extra so no need to specify it here too.